src/main/java/storagesystem/model/ReservationHandler.java

```diff
@@ -116,7 +115,7 @@ private boolean isObjectReservedBetween(IReservable object, Interval interval) {
      *
      * @param borrower The party which want to borrow object
      * @param interval The interval in which to reserve the object.
-     * @param object The object which is to be reserved.
+     * @param object   The object which is to be reserved.
      */
     public void createReservation(IBorrower borrower, Interval interval, IReservable object) {
         if (!isObjectReservedBetween(object, interval)) {
```

Comment:
Shouldn't it be checked before this method is called? Otherwise this method does more than its name says.